accord-core/build.gradle

```diff
@@ -56,15 +56,32 @@ task testJar(type: Jar, dependsOn: testClasses) {
 publishing {
   publications {
     mavenJava(MavenPublication) {
-      artifactId = 'accord'
+      artifactId = accord_artifactId
       pom {
+        name = 'Apache Cassandra Accord'
+        description = 'General purpose transactions library using a leaderless consensus protocol to have highly available transactions'
+        url = 'https://cwiki.apache.org/confluence/display/CASSANDRA/CEP-15%3A+General+Purpose+Transactions'
```

Comment:
I guess we don't have a README for https://github.com/apache/cassandra-accord, so this is probably the most useful link for now...

Reply:
@belliottsmith any feedback on best URL to use?  This should be allowed to change over time, felt the CEP wiki was the best for now

Reply:
Makes sense yeah